docs/CHANGELOG.rst

```diff
@@ -6,6 +6,14 @@ All notable changes to this project are documented in this file.
 This project adheres to `Semantic Versioning <http://semver.org/>`_.
 
 
+==========
+Unreleased
+==========
+
+Changed
+-------
+- Change to unbuffered stdout/stdin
```

Comment:
When talking on Slack I got this wrong: it is stdout and stderr. Can you please change stdin to stderr ?